validator/src/main/java/com/google/daq/mqtt/registrar/Registrar.java

```diff
@@ -882,6 +882,7 @@ private void sendUpdateMessages(LocalDevice localDevice) {
     }
   }
 
+  //TODO this is now broken if device uses a file rather than automatically generated config
```

Comment:
Space after //, and colon after TODO... so "// TODO: " -- just for consistency.

Reply:
Should these bits  be removed or fixed to work with generic JSON types? because they also won't work with any downgraded messages I think. They're also redundant now that UDMIS handles the actual config rather than direct through the IoT Core API